zproject/prod_settings_template.py

```diff
@@ -340,6 +341,36 @@
 #
 # SOCIAL_AUTH_SUBDOMAIN = 'auth'
 
+########
+## Generic OpenID Connect.
+##
+## (1) Choose the name for this authentication method and configure it below
+## as `SOCIAL_AUTH_GENERIC_OIDC_NAME`.
+## It doesn't show in any user-facing strings, but the endpoint for the Return URL is formed based
+## on it (`/complete/example/` if the name is "example") and the names of the settings you'll configure
+## are generated based on it.
+# SOCIAL_AUTH_GENERIC_OIDC_NAME = "example"
```

Comment:
Generally we've been moving towards having the documentation in ReadTheDocs rather than in comments, because the latter are much more annoying to update.  Maybe we want to move basically all of this content there?

Reply:
I feel it's easier to explain and phrase things via "Below is a setting for <this>" where the user has a visible, available to uncomment, example of how the setting is set. But yeah, ReadTheDocs is ultimately better. Converting the instructions to ReadTheDocs will take a bit more rewriting, so I left thing in prod_settings_template for now and move it when reworking this current first-draft doc.

Reply:
Yeah, that's reasonable, especially since we're potentially changing how the configuration works.